content/10.peptide-ionization.md

```diff
@@ -1,7 +1,21 @@
 ## Peptide Ionization {.page_break_before}
-
-The 2002 Nobel Prize in Chemistry was awarded to partially to John Fenn and Koichi Tanaka "for their development of soft desorption ionisation methods for mass spectrometric analyses of biological macromolecules" [@URL:https://www.nobelprize.org/prizes/chemistry/2002/summary/].
+Before MALDI and ESI[@DOI:10.1126/science.2675315], the so-called soft-ionization techniques were developed, the analysis of peptides was a challenge for mass spectrometry. 
+It was a big ask to introduce ionized peptides using a hard ionization technique (like fast atom bombardment) into the mass spectrometer, without destroying or breaking them.
+The soft ionization tehniques however, revolutionized the proteomics field and it became possible to routinely ionize and analyze peptides using MALDI and ESI techniques at high-throughput scale. 
+Such was the impact of these two techniques that the 2002 Nobel Prize in Chemistry was co-awarded to John Fenn (ESI) and Koichi Tanaka (MALDI) "for their development of soft desorption ionisation methods for mass spectrometric analyses of biological macromolecules". [@URL:https://www.nobelprize.org/prizes/chemistry/2002/summary/].
 
 ### MALDI
+In this method, the in-gel digested protein sample is co-crystallized with a matrix - a volatile, low molecular-weight, aromatic compound (like cyno-hydroxycinnamic acid).
+Subsequently, the analyte is ionized by Matrix-assisted laser desorption ionization (MALDI), in which it is subjected to a 337nm LASER.
+This laser energy is just about right to ionize the matrix and peptides without breaking them.
+The matrix and co-crystallized sample generate plumes, and the volatile matrix imparts its protons to the peptides as it gets ionized first.
+The weak acidic conditions used as well as the acidic nature of the matrix allows easy exchange of protons for the peptides to get ionized.
+These ionized peptides generally form the metastable ions, most of them will fragment quickly.
+However, it can take several milliseconds and the mass spectrometry analysis can be performed before this time.
 
 ### Electrospray Ionization
+In the electrospray ionization, reverse-phase liquid-chromatography setup is usually connected to an online nano-ESI source.
+A high voltage is applied between the spraying needle and the mass spectrometer.
+Through this needle, a fine mist of analyte as liquid droplets form at the needle.
+Due to voltage and pressure, the droplet bursts into plumes of ions carrying protons from the liquid phase of the chromatography to impart ions to the analytes.
+The liquid phase is generally kept acidic to impart protons easily to the analytes.
```

Comment:
should mention taylor cone and coulombic explosion somewhere in this part, These look like a good overviews to cite: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3348530/ 
https://doi.org/10.1002/(SICI)1096-9888(199707)32:7%3C677::AID-JMS536%3E3.0.CO;2-G

Reply:
Looks fine to cite these two.